### PR TITLE
Update print statements to make it Py3 compatible

### PR DESCRIPTION
**Pull request by captainmalloc**
Just a small update to make it run with Python 3

### CODE REVIEW

File: cover2cover.py [hunk 1]
```diff
@@ -142,12 +142,12 @@ def jacoco2cobertura(filename, source_roots):
 
     into = ET.Element('coverage')
     convert_root(root, into, source_roots)
-    print '<?xml version="1.0" ?>'
-    print ET.tostring(into)
+    print('<?xml version="1.0" ?>')
+    print(ET.tostring(into))
```
Comment (EpicEric) [inline cover2cover.py]: They've changed the behavior of `ET.tostring` in Python3. This should be:

```python3
print(ET.tostring(into, encoding='unicode'))
```

**Review by EpicEric — CHANGES_REQUESTED** [review]

**Review by EpicEric — APPROVED** [review]

**Comment by MikeDombo** [comment]
I'd love to use this script, can this get merged?

**Comment by kayman-mk** [comment]
@rix0rrr Any chance to get this PR merged?

**Comment by rix0rrr** [comment]
Fixed by merging another PR

**Pull request closed without merging**